File: tests/data-clumps/test-cases/parameter-parameter/simple/negative/java/source/AppointmentScheduler.java
public class AppointmentScheduler {
    public void schedule(int patientId, int doctorId, boolean requiresFollowUp) {
        System.out.println("Scheduling appointment for patient " + patientId + " with doctor " + doctorId
                + " (requires follow-up: " + requiresFollowUp + ")");
    }
}

File: tests/data-clumps/test-cases/refactoring/parameter-parameter/simple/typescript/source-expected/AppointmentScheduler.ts
import { PatientIdDoctorIdRequiresFollowUpParams } from "./PatientIdDoctorIdRequiresFollowUpParams";

export class AppointmentScheduler {
  schedule(params: PatientIdDoctorIdRequiresFollowUpParams): void {
      const { patientId, doctorId, requiresFollowUp } = params;
    console.log(`Scheduling appointment for patient ${patientId} with doctor ${doctorId} (requires follow-up: ${requiresFollowUp})`);
  }
}
